.github/workflows/release-python.yml

```diff
@@ -0,0 +1,90 @@
+name: Release Python
+
+on:
+  push:
+    tags:
+      - python/v*
+
+jobs:
+  macos:
+    runs-on: macos-11
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        if: matrix.python-version != '3.7'
+        with:
+          toolchain: stable
+          target: aarch64-apple-darwin
+          profile: minimal
+          default: true
+      - name: Build wheels - x86_64
+        # Python 3.7 does not support aarch64 so we only build it for x86_64
+        if: matrix.python-version == '3.7'
+        uses: messense/maturin-action@v1
+        with:
+          target: x86_64
+          args: -i python --release --manifest-path crates/python/Cargo.toml --out dist --sdist
+      - name: Build wheels - universal2
+        # universal2 supports both x86_64 and aarch64 so every Python > 3.7 only needs this wheel
+        if: matrix.python-version != '3.7'
+        uses: messense/maturin-action@v1
+        with:
+          args: -i python --release --universal2 --manifest-path crates/python/Cargo.toml --out dist
+      - name: Install wheel
+        run: |
+          pip install qcs-sdk-python --no-index --find-links dist --force-reinstall
```

Comment:
Looking at this again, are we sure this is installing the new wheel? I don't see any file path being passed in.

I'm not terribly familiar with how pip works, so I suppose it could be finding it automatically, but I couldn't find a CI job to verify with.

Reply:
Yes. The wheels are output to the `dist` directory and the `--no-index --find-links dist` flag is whats telling pip to look in that directory for the wheels and not the package index.